modelbaker/utils/db_utils.py

```diff
@@ -122,7 +122,9 @@ def get_db_connector(configuration):
 
     db_factory = db_simple_factory.create_factory(configuration.tool)
     config_manager = db_factory.get_db_command_config_manager(configuration)
-    uri_string = config_manager.get_uri(configuration.db_use_super_login)
+    uri_string = config_manager.get_uri(
+        configuration.db_use_super_login, QgsApplication.userLoginName()
```

Comment:
Since there's a `qgis` parameter in between in `get_uri()`, shouldn't this be

`su: configuration.db_use_super_login, fallback_user:QgsApplication.userLoginName()`

instead?

Reply:
oops